frontend/src/scenes/max/components/StageNotebooksDisplay.tsx

```diff
@@ -0,0 +1,79 @@
+import { IconCheck, IconNotebook } from '@posthog/icons'
+import { LemonButton } from '@posthog/lemon-ui'
+
+import { IconOpenInNew } from 'lib/lemon-ui/icons'
+import { NotebookTarget } from 'scenes/notebooks/types'
+
+import { openNotebook } from '~/models/notebooksModel'
+import { DeepResearchNotebookInfo } from '~/types'
+
+interface StageNotebooksDisplayProps {
+    stageNotebooks: DeepResearchNotebookInfo[]
+}
+
+const STAGE_DISPLAY_NAMES: Record<string, string> = {
+    planning: 'Planning',
+    notebook_planning: 'Planning',
+    report: 'Final Report',
+}
+
+const STAGE_DESCRIPTIONS: Record<string, string> = {
+    planning: 'Initial research plan and objectives',
+    notebook_planning: 'Initial research plan and objectives',
+    report: 'Comprehensive analysis and findings',
+}
+
+export function StageNotebooksDisplay({ stageNotebooks }: StageNotebooksDisplayProps): JSX.Element {
+    const handleOpenNotebook = (notebookId: string): void => {
+        openNotebook(notebookId, NotebookTarget.Scene)
+    }
+
+    if (!stageNotebooks.length) {
+        return <></>
+    }
+
+    return (
+        <div className="bg-bg-light border border-border rounded-lg p-4 space-y-3">
+            <div className="flex items-center gap-2">
+                <IconCheck className="text-success size-4" />
+                <h4 className="text-sm font-semibold m-0">Deep Research Complete</h4>
```

Comment:
This doesn't work for session summaries, maybe `NotebookUpdateMessage` can have its own status `type="deep_research" | "session_summaries"`, and this maps to different title/subtitles in the frontend.